src/test/unit/helperFunctions/test_yara_binary_search.py

```diff
@@ -3,30 +3,90 @@
 from helperFunctions import yara_binary_search, fileSystem
 from os import path
 
+from helperFunctions.config import get_config_for_testing
+from helperFunctions.yara_binary_search import YaraRuleError
+
+
+TEST_FILE_1 = 'binary_search_test'
+TEST_FILE_2 = 'binary_search_test_2'
+TEST_FILE_3 = 'binary_search_test_3'
+
+
+class MockCommonDbInterface:
```

Comment:
Is it possible to use one of the existing mock interfaces. I would rather enjoy it if we would reduce the number of hard coded mocks instead of increasing it ; )